spec/intl.html

```diff
@@ -72,53 +124,66 @@ <h1><a href="https://tc39.es/ecma402/#sec-isvalidtimezonename">IsValidTimeZoneNa
           1. Return *false*.
         </emu-alg>
       </emu-clause>
-    </del>
-
-    <emu-clause id="sup-canonicalizetimezonename" type="abstract operation">
-      <h1>
-        CanonicalizeTimeZoneName (
-          _timeZone_: a String value that is <del>a valid</del><ins>an available</ins> time zone name as verified by <del>IsValidTimeZoneName</del><ins>IsAvailableTimeZoneName</ins>,
-        )
-      </h1>
-      <dl class="header">
-        <dt>description</dt>
-        <dd>It returns the canonical and case-regularized form of _timeZone_.</dd>
-        <dt>redefinition</dt>
-        <dd>true</dd>
-      </dl>
 
-      <emu-alg>
-        1. Let _ianaTimeZone_ be the String value of the Zone or Link name of the IANA Time Zone Database that is an ASCII-case-insensitive match of _timeZone_.
-        1. If _ianaTimeZone_ is a Link name, let _ianaTimeZone_ be the String value of the corresponding Zone name as specified in the file <code>backward</code> of the IANA Time Zone Database.
-        1. If _ianaTimeZone_ is *"Etc/UTC"* or *"Etc/GMT"*, return *"UTC"*.
-        1. Return _ianaTimeZone_.
-      </emu-alg>
+      <emu-clause id="sup-canonicalizetimezonename" type="abstract operation">
+        <h1>
+          CanonicalizeTimeZoneName (
+            _timeZone_: a String value that is a valid time zone name as verified by IsValidTimeZoneName,
+          )
```

Comment:
```suggestion
          ): a String
```

Reply:
This is inside a `<del>` block. It's an AO that will be removed  from ECMA-402. So I'll plan to leave it the text here as-is to better match the original ECMA-402 text.